eclipse/sw360
# fix(ui): Fixed Generate License Info, Clearing Report and Source Code Download with/without subproject and showAll/onlyApproved filter

> Fixed Generate License Info, Clearing Report and Source Code Download with/without subproject and showAll/onlyApproved filter
> * Which issue is this pull request belonging to and how is it solving it? (*Refer to issue here*)
> * Did you add or update any new dependencies that are required for your change? - No

Issue: 

### How To Test?
>  Generate License Info, Clearing Report and Source Code Download with/without subproject and showAll/onlyApproved
>  - Validate downloaded content.
>  - Reload page and validate attachmentUsage/selection is saved.
>  - Toggle between `showAll/onlyApproved` view, verify list content is ok
>     - (from discussion) - for download from `only approved` tab. attachementUsage for selected approved attachment + existing(in DB) attachementUsage for non-approved attachment , will be saved.

> Have you implemented any additional tests?

### Checklist
Must:

Signed-off-by: Jaideep Palit <jaideep.palit@siemens.com>
